.github/workflows/build-test-contracts-common.yaml

```diff
@@ -20,7 +20,7 @@ on:
 name: Clippy & fmt
 
 env:
-  RUST_VERSION: "1.65"
+  RUST_VERSION: "1.73"
```

Comment:
So this version is the minimal supported rust version.

We should not be just bumping it without a good reason.

And when we bump it it must be accompanied by a minor version bump, and reflected in the changelog.

Reply:
Package `bumpalo` requires minimal supported rust version `1.73`.
https://github.com/Concordium/concordium-base/actions/runs/8155571275/job/22291370938